### PR TITLE
api: add docstring to endpoints with additional methods

### PR DESCRIPTION
**Pull request by mwasilew**
Signed-off-by: Milosz Wasilewski <milosz.wasilewski@linaro.org>

### CODE REVIEW

**Comment by mwasilew** [comment]
This addresses the issue raised in #712

File: squad/api/rest.py [hunk 1]
```diff
@@ -430,6 +430,32 @@ class ProjectViewSet(viewsets.ModelViewSet):
...
+
+     * `api/projects/<id>/suites` GET
+
+        List of test suite names available in this project
+
+     * `api/projects/<id>/tests` GET
+
+        List of test names available in this project
+
+     * `api/projects/<id>/test_results` GET
```
Comment (chaws) [inline squad/api/rest.py:448]: This exposed a bug when no `test_name` is passed in the querystring: https://github.com/Linaro/squad/issues/729

File: squad/api/rest.py [hunk 2]
```diff
@@ -896,6 +956,13 @@ class TestNameSerializer(serializers.BaseSerializer):
 
 
 class SuiteViewSet(viewsets.ModelViewSet):
+    """
+    Additional actions:
+
+     * `api/testruns/<id>/testnames` GET
```
Comment (chaws) [inline squad/api/rest.py:962]: api/testruns/<id>/testnames => api/suites/<id>/testnames

**Comment by chaws** [comment]
Looks good, I'll just wait for the adjustment then I can merge it.

**Pull request merged**